be/src/olap/storage_engine.cpp

```diff
@@ -357,6 +357,12 @@ OLAPStatus StorageEngine::get_all_data_dir_info(vector<DataDirInfo>* data_dir_in
     size_t tablet_count = 0;
     _tablet_manager->update_root_path_info(&path_map, &tablet_count);
 
+    // update trash used capacity
+    for (auto& path : path_map) {
+        std::filesystem::path trash_path(path.first + TRASH_PREFIX);
```

Comment:
I recommend not using the right value here to construct std::filesystem::path directly because it might trigger one of gcc 10.1.0

Reply:
fixed.